File: c/Prime_upto_n.c
#include <stdio.h>
#include <stdlib.h>
#include<time.h>
#include<math.h>

int isPrime(int n)
{
    for(int i=2;i<=sqrt(n);i++)
        if (n%i==0)
            return i;
    return n;
}

int main()
{
    int n;
    clock_t t0,t1;
    printf("Enter The Upper Bound : ");
    scanf("%d",&n);
    t0=clock();
    printf("The Prime Numbers Upto %d are \n",n);
    for(int i=2;i<=n;i++)
        if (isPrime(i)==i)
            printf("%d\n",i);
    t1=clock();
    printf("Time : %f",(t1-t0)/(double)CLOCKS_PER_SEC);
    return 0;
}

File: java/prime_upto_n.java
package algebra;
import java.util.Scanner;
public class prime_upto_n {
		private static Scanner s;
		int i,j,p=0;
		void prime(int n)
		{
				System.out.println("Prime numbers from 1 to "+n+" are");
				for(i=2;i<=n;i++)
				{
					for(j=2;j<=i;j++)
					{
						if(i%j==0)
							break;
					}
					if(i==j)
					{
						System.out.println(i);
						p++;
					}	
				}
				System.out.println("Total : "+p);
		}
		public static void main(String args[])
		{
			prime_upto_n a=new prime_upto_n();
			s=new Scanner(System.in);
			System.out.print("Enter the value of n : ");
			int n=s.nextInt();
			a.prime(n);
		}
}
